tests/helper/apiHelper.js

```diff
@@ -10,7 +10,16 @@ class ApiHelper extends Helper {
         user_id: 1,
         product_tour_completed: true,
         alerting_tour_completed: true,
-        snoozed_pmm_version: '3.1.0',
+        snoozed_pmm_version: '',
+      }),
+    }));
+
+    await page.route('**/v1/server/updates?force=true', (route) => route.fulfill({
```

Comment:
This will change in the future to be `force=false` (https://perconadev.atlassian.net/browse/PMM-13453)

Might be good to mock both variants.

Reply:
Change regex to allow different values for the force